doc/developers/advanced_installation.rst

```diff
@@ -46,7 +46,8 @@ Scikit-learn requires:
 
 Building Scikit-learn also requires
 
-- Cython >=0.23 
+- Cython >=0.23
```

Comment:
Not sure what is the diff there but you might want to remove it to avoid conflict in #13064